src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
+     :active (apply + (map count-map ["Submitted" "Running" "Aborting"]))
+     :terminal (apply + (map count-map ["Failed" "Succeeded" "Aborted"]))}))
+
+(react/defc GlobalSubmissionStatus
+  {:render
+   (fn [{:keys [state]}]
+     (let [status-error (:status-error @state)
+           status-counts (:status-counts @state)]
+       (cond
+         status-error [:span {} [:br {}] "Submissions: " status-error]
+         status-counts [:span {} [:br {}] "Submissions: " (:queued status-counts) " Queued; " (:active status-counts) " Active"]
```

Comment:
After looking at what's rendered in Chrome's DOM inspector, I'm fine with keeping this as a span; it's not a big deal.

However, I suggest a minor change from 
```
[:span {} [:br {}] "Submissions: " (:queued status-counts) " Queued; " (:active status-counts) " Active"]
```
to
```
[:span {} [:br {}] (str "Submissions: " (:queued status-counts) " Queued; " (:active status-counts) " Active")]
```
to keep the DOM as light as possible. The former produces:
```
<span data-reactid=".2.0.0.0.0.1.1">Submissions: </span>
<span data-reactid=".2.0.0.0.0.1.2">0</span>
<span data-reactid=".2.0.0.0.0.1.3"> Queued; </span>
<span data-reactid=".2.0.0.0.0.1.4">2</span>
<span data-reactid=".2.0.0.0.0.1.5"> Active</span>
```
while the latter produces:
```
<span data-reactid=".1.0.0.0.0.1.1">Submissions: 0 Queued; 2 Active</span>
```